<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QWidget" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/chart</normaloff>:/icons/chart</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblStartTime">
       <property name="text">
        <string>Start Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="startTime">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to change the start time of the chart. Click the refresh button to update.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2013</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblEndTime">
       <property name="text">
        <string>End Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="endTime">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to change the end time of the chart. Click the refresh button to update.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2013</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="time">
        <time>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pbRefresh">
       <property name="whatsThis">
        <string>Reloads the chart. Click this if you have changed the time range to display new data.</string>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/refresh</normaloff>:/icons/refresh</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="divRefresh">
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbYLock">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>If any Y axis is selected all Y axes will scale and pan at the same time</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;When this checkbox is checked then all Y axes will be selected at once. This allows all Y axes to be scaled/panned together.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Lock Y Axes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbXLock">
       <property name="text">
        <string>Lock X Axes</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="YLockDiv">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="whatsThis">
        <string>Allows saving the chart in PNG, JPG, BMP and PDF formats.</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCustomPlot" name="chart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="whatsThis">
      <string>Use the mouse wheel to zoom, drag to pan. Double click on the axes to limit pan and zoom.</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSave_as_PDF">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/export-chart/pdf</normaloff>:/icons/export-chart/pdf</iconset>
   </property>
   <property name="text">
    <string>Save as PDF...</string>
   </property>
  </action>
  <action name="actionSave_as_PostScript">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/export-chart/postscript</normaloff>:/icons/export-chart/postscript</iconset>
   </property>
   <property name="text">
    <string>Save as PostScript...</string>
   </property>
  </action>
  <action name="actionSave_as_PNG">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/export-chart/png</normaloff>:/icons/export-chart/png</iconset>
   </property>
   <property name="text">
    <string>Save as PNG...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>charts/qcp/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
